Header header

GimbalState[] gimbal_state
GimbalCmd[] gimbal_cmd
sensor_msgs/Imu[] imu
detection_msgs/BoundingBoxes[] bounding_boxes

float32 gimbal_yaw
float32 gimbal_pitch
int32 gimbal_state_machine
int32 zoom_in_state
float32 drone_pitch
float32 gimbal_yaw_cmd
float32 gimbal_pitch_cmd